Modules/Devices/src/Devices.Application/Devices/Queries/GetActiveDevice/Handler.cs

```diff
@@ -15,24 +15,20 @@ public class Handler : IRequestHandler<GetActiveDeviceQuery, DeviceDTO>
     private readonly IDevicesDbContext _dbContext;
     private readonly IMapper _mapper;
     private readonly IUserContext _userContext;
+    private readonly IDevicesRepository _devicesRepository;
 
-    public Handler(IDevicesDbContext dbContext, IMapper mapper, IUserContext userContext)
+    public Handler(IDevicesDbContext dbContext, IMapper mapper, IUserContext userContext, IDevicesRepository devicesRepository)
```

Comment:
```suggestion
    public Handler(IMapper mapper, IUserContext userContext, IDevicesRepository devicesRepository)
```

Delete the `IDevicesDbContext` interface to see if there are any other references

Reply:
This interface is used in the `Modules\Devices\src\Devices.Application\ChallengeValidator.cs` file.